static-build/cloud-hypervisor/build-static-clh.sh

```diff
@@ -36,6 +36,6 @@ repo_dir="${repo_dir//.git}"
 cd "${repo_dir}"
 git fetch || true
 git checkout "${cloud_hypervisor_version}"
-"${script_dir}/docker-build/build.sh"
+./scripts/dev_cli.sh build --release --libc musl
```

Comment:
I dont see the dev_cli.sh file in this repo. Are we adding it in this PR?
Also, prefer to use ${script_dir} instead os `./scripts` since this script could be invoked from another directory.

Reply:
This patch reuses the building script from clh repo: https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/scripts/dev_cli.sh. It relies on the container image created/maintained as a part of the clh project. I think it is better to leverage their efforts, instead of maintaining another docker image to build clh in Kata.

Also, using the hard-coded path to `/scripts/dev_cli.sh` should be good given the context of this `build-static-clh.sh` . I assume this `build-static-clh.sh` will work independent of where it is invoked. Please correct me if I am wrong. Any other thoughts, please also let me know. :)

Reply:
/cc @sboeuf

Reply:
This seems reasonable to me. The relative path is within the repo we just pulled -- it's handled by the cd $repo_dir.

We just have to hope that they don't reorganize within their repos structure.  If they do, we'd need to redo our build logic anyway.  lgtm.